tamr_toolbox/utils/operation.py

```diff
@@ -128,3 +126,31 @@ def wait(
             return operation
         operation = operation.poll()
     raise TimeoutError(f"Waiting for operation took longer than {timeout_seconds} seconds.")
+
+
+def monitor(
+    operation: Operation, *, poll_interval_seconds: int = 1, timeout_seconds: int = 300,
```

Comment:
timeout parameters should have type `float` as in `notification.emails.monitor_job`
```suggestion
    operation: Operation, *, poll_interval_seconds: float = 1, timeout_seconds: float = 300,
```

Reply:
Updated